.github/workflows/publish-javadoc.yml

```diff
@@ -17,18 +21,12 @@ jobs:
       - name: Extract Robolectric version
         id: robolectric_version
         run: |
-          authorAssociation="${{ github.event.issue.author_association }}"
+          if [[ "${{ inputs.version }}" =~ ^(([0-9]+\.[0-9]+)(\.[0-9]+)?)$ ]]; then
+            robolectricMinorVersion="${BASH_REMATCH[2]}"
+            robolectricPatchVersion="${BASH_REMATCH[1]}"
 
-          if [[ "$authorAssociation" == "COLLABORATOR" || "$authorAssociation" == "MEMBER" || "$authorAssociation" == "OWNER" ]]; then
```

Comment:
Do we need to keep author verification?

Reply:
I don't think so. The workflow can only be triggered from the `Actions` tab, so only maintainers can run it.